e2e/rbd_helper.go

```diff
@@ -30,6 +30,40 @@ func rbdOptions(pool string) string {
 	return "--pool=" + pool
 }
 
+type errGettingFsid string
+type errListingRBDNs string
+type errCreatingRBDNs string
+type errMsg string
+type getOmapVal string
+type errReadPassphrase string
```

Comment:
Instead of creating new types, it maybe you can create new static errors like

```golang
var (
    errGetFsid = errors.New("error getting fsid")
)
```

And return that in functions, or use

```golang
return nil, fmt.Errorf("could not fund fsid for %q: %w", rbdVol, errGetFsid)
```

Having new types for all error strings seems a little much to me.

Reply:
@nixpanic in this case suggested by you, we will miss the actual error string that was printed by `stdErr` , hence, I see creating new types will be better.